shogun-lib/src/main/java/de/terrestris/shogun/lib/controller/GroupController.java

```diff
@@ -93,4 +98,9 @@ public List<User> getGroupMembers(@PathVariable("id") String keycloakId) {
         }
     }
 
+    @PostMapping("/createFromProvider")
+    @ResponseStatus(HttpStatus.OK)
+    public void createFromProvider() {
+        groupProviderService.createAllGroups();
+    }
```

Comment:
Can we add some notes for the OpenAPI spec? The controllers in the [BaseController](https://github.com/terrestris/shogun/blob/main/shogun-lib/src/main/java/de/terrestris/shogun/lib/controller/BaseController.java) might be a good inspiration.

And can we also add some error handling?